README.md

```diff
@@ -173,7 +163,7 @@ You need valid developer credentials for `cmsFronts` and `workflow`.
 You can get keys temporary keys from `janus`. Note that the recommended way to get Janus credentials for *facia-tool* is to use feria and run:
 
 ```
-$ feria cmsFronts && feria --access s3-read workflow && feria --access sqs-consumer frontend
```

Comment:
Even when we don't write anymore to the other bucket we still need to send messages to sqs queue to let facia press know that a change has happened.